libbeat/cmd/instance/beat.go

```diff
@@ -138,6 +141,18 @@ type beatConfig struct {
 	TimestampPrecision *config.C `config:"timestamp"`
 }
 
+type certReloadConfig struct {
+	tlscommon.Config `config:",inline" yaml:",inline"`
+	Reload           cfgfile.Reload `config:"exit_on_cert_change" yaml:"exit_on_cert_change"`
+}
+
+func (c certReloadConfig) Validate() error {
+	if c.Reload.Period < time.Second {
+		return errors.New("'exit_on_cert_change.period' must be equal or greather than 1s")
+	}
+	return nil
+}
```

Comment:
Is it called automatically by `.Unpack` because of some interface convention? I don't see it explicitly used anywhere.

Reply:
Yes, it is.